slist.cabal

```
@@ -11,23 +11,23 @@ license-file:        LICENSE
 author:              Veronika Romashkina
 maintainer:          Kowainik <xrom.xkov@gmail.com>
 copyright:           2019-2020 Veronika Romashkina
-                     2020 Kowainik
+                     2020-2021 Kowainik
 category:            Data Structures, List
 build-type:          Simple
 extra-doc-files:     README.md
                    , CHANGELOG.md
 tested-with:         GHC == 8.2.2
                    , GHC == 8.4.4
                    , GHC == 8.6.5
-                   , GHC == 8.8.3
-                   , GHC == 8.10.1
+                   , GHC == 8.8.4
+                   , GHC == 8.10.4
```

Let's add GHC-9.0 here as well 🙂 
```suggestion
                   , GHC == 8.10.4
                   , GHC == 9.0.1
```